pack/pack.js

```diff
@@ -88,6 +81,6 @@ function usage() {
     utils.logInfo('  pack.js --usage', COLOR.magenta);
```

Comment:
pack.js arguments have changed - to generate all packages one should pass  `--cli=forceios,forcedroid,forcehybrid` instead of `--os=ios,android`.